docs/build-on-linea/tooling/oracles/chainlink.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Chainlink
+---
+
+## Chainlink
+
+Chainlink is the industry standard for building, accessing, and selling oracle services needed to power hybrid smart contracts on any blockchain. Chainlink oracle networks provide smart contracts with a way to reliably connect to any external API and leverage secure offchain computations for enabling feature-rich applications. Chainlink currently secures tens of billions of dollars across DeFi, insurance, gaming, and other major industries, and offers global enterprises and leading data providers a universal gateway to all blockchains.
+
+
+## Chainlink Products
+
+1. [Data Streams](https://docs.chain.link/data-streams): Provides high-frequency market data and enables automated trades.
```

Comment:
I think we only have data feeds on Linea? I'll reach out to check with them + if they need any additions

Reply:
yeah, it's just one of the products they offer on their page. Wasn't specifically saying that it is available for Linea